EnvironmentSimulator/Modules/CommonMini/CommonMini.cpp

```diff
@@ -235,8 +235,6 @@ double PointToLineDistance2DSigned(double px, double py, double lx0, double ly0,
 {
 	double l0x = lx1 - lx0;
 	double l0y = ly1 - ly0;
-	double l1x = px - lx0;
```

Comment:
I see now that l0x and l0y could have been used in the cross product call. But that's an old mistake, and not a big deal.